CRM/Core/Payment.php

```diff
@@ -713,6 +713,10 @@ public function getPaymentFormFieldsMetadata() {
             'rule_parameters' => TRUE,
           ),
         ),
+        'aria-labels' => array(
+          'credit_card_exp_date_M' => ts('Expiration Month'),
+          'credit_card_exp_date_Y' => ts('Expiration Year'),
+        ),
```

Comment:
@monishdeb I see you are using javascript to add some labels. Is that because those form elements are not built with Quickform?

Reply:
@monishdeb maybe we should patch QF to add the aria-label attrs to date sub-elements, concatenating the main label with the name of the element. So if the date element is labeled "Credit card expiration" then we patch QF to automatically make

    'aria-label' => $mainLabel . ' ' . ts('month') // Credit card expiration month